pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.23</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.73.3</jenkins.version>
+        <jenkins.version>2.145-rc27341.296ad9122b93</jenkins.version>
```

Comment:
As a courtesy please indicate the upstream PR in a comment:

```xml
<jenkins.version>2.145-rc27341.296ad9122b93</jenkins.version> <!-- TODO https://github.com/jenkinsci/jenkins/pull/3611 -->
```

Reply:
Thanks for the hint. Next time I'll take care. For this time I'll change the dependency to 2.145 assuming it won't need the TODO.